doc/Manuals/ug-cp_x-road_v6_configuration_proxy_manual.md

```diff
@@ -1,31 +1,33 @@
 # X-Road: Configuration Proxy Manual
 
-Version: 2.12  
+Version: 2.13  
```

Comment:
The manual doesn't explain what needs to be done to switch the global configuration sign key type from RSA to EC. It should be explained what's the default key type and how it can be changed. Also, the documentation should explain how changing the key type may affect backwards compatibility and interoperability between two X-Road instances in a federated setup.

Reply:
Updated